simplesecurity/plugins.py

```diff
@@ -1,326 +1,460 @@
-"""Add plugins here.
```

Comment:
Could you convert to tabs please as it's making review very challenging?

Reply:
I will look into it! If you know any tool that does this cleanly, let me know :)